components/post_view/post_edited_indicator/post_edited_indicator.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import Icon from '@mattermost/compass-components/foundations/icon/Icon';
+
+interface Props {
+    postId?: string | null;
```

Comment:
Any strong reason to null this value? Can it be just undefined?

Reply:
i am setting a default value and I am not a big fan of setting values manually to undefined like this:

```typescript
const Component = ({ param = undefined, ... }) => { ... }
```

Reply:
although I changed the structure a bit in the following commits, so changing this is now totally fine